libraries/shared/test_data/test_helpers.go

```diff
@@ -88,3 +89,30 @@ func getRandomAddress() string {
 	stringHash := hashToPrefixedString(hash)
 	return stringHash[:42]
 }
+
+/// DescribeAValidCheckedHeadersModelWithSchema is provided so that
```

Comment:
seems like this comment is lying about the function name 🕵️

Reply:
Well at one time it was right. It's lying about how to use it.